package.json

```diff
@@ -57,10 +57,10 @@
   },
   "homepage": "https://github.com/thepracticaldev/dev.to#readme",
   "devDependencies": {
-    "@storybook/addon-actions": "^5.1.9",
-    "@storybook/addon-links": "^5.1.9",
-    "@storybook/addons": "^5.1.8",
-    "@storybook/react": "^3.4.11",
+    "@storybook/addon-actions": "3.4.11",
```

Comment:
This PR looks like nothing... it is. However, I was scratching my head for a whole day why downgrading didn't work. I finally literally copied the entire package.json version from the last working Storybook in the repo and it still wasn't working. And then it clicked. I've had this issue before. I had to delete the yarn.lock file and completely regenerate it, to have the  proper deps in the lock file. 🙃

Reply:
what a doozy